generatedocs/compile/testdata/base/data/actions/changesheet/description.md

```diff
@@ -0,0 +1,25 @@
+## ChangeSheet action
+
+Change to a new sheet, unsubscribe to the currently subscribed objects, and subscribe to all objects on the new sheet.
+
+The action supports getting data from the following objects:
+
+* Listbox
+* Filter pane
+* Bar chart
+* Scatter plot
+* Map (only the first layer)
+* Combo chart
+* Table
+* Pivot table
+* Line chart
+* Pie chart
+* Tree map
+* Text-Image
+* KPI
+* Gauge
+* Box plot
+* Distribution plot
+* Histogram
+* Auto chart (including any support generated visualization from this list)
+* Waterfall chart
```

Comment:
Not related to this PR, but this information does not seem to belong here IMO. It doesn't seem justified to have an extra listing of every single object for this particular action.

Reply:
Can't tell really. Do we have this information somewhere else?

Will keep it in the testing-set.

Reply:
Well we have the limitations: https://github.com/qlik-oss/gopherciser/blob/master/docs/README.md#limitations Which makes more sense to present, so I don't know why we'd also need this full list. Seems it'd just make it more difficult to maintain. I'll leave it up to @DnlLrssn

Reply:
it's old docs that's there since before limitations section. In the beginning we only supported few objects and added them to this list. Can I agree it should probably be rewritten with a link to the limitations by now.

Reply:
Yes, it fits better in limitations., but do we really want links to readme in the in-memory docs?